cmd/aro/operator.go

```diff
@@ -108,6 +109,12 @@ func operator(ctx context.Context, log *logrus.Entry) error {
 		clustercli, arocli, role, deploymentMode)).SetupWithManager(mgr); err != nil {
 		return fmt.Errorf("unable to create controller InternetChecker: %v", err)
 	}
+
+	if err = (routefix.NewReconciler(
```

Comment:
controller should run on master only - put under `if role == pkgoperator.RoleMaster` block above